### PR TITLE
fix: use all parcels to determine connections

### PR DESCRIPTION
**Pull request by NicoSantangelo**
@cazala ¯\\_(ツ)_/¯

### CODE REVIEW

File: shared/parcel.js [hunk 1]
```diff
@@ -20,21 +20,14 @@ export function isParcel(asset) {
   )
 }
 
-export function toParcelObject(
-  parcelsArray,
-  prevParcels = [],
-  normalize = true,
-  redoConnections = false
```
Comment (nachomazzara) [inline shared/parcel.js]: If we remove the `redoConnections` we will lose the _feature_ of a confirmed tx of removing a parcel from an Estate

Comment (cazala) [reply shared/parcel.js]: That's taken care by this line: https://github.com/decentraland/marketplace/pull/440/files#diff-fa45755e5e2585857eb74278da106dd6R51

File: shared/parcel.js [hunk 2]
```diff
@@ -21,14 +21,11 @@ export function isParcel(asset) {
...
 export function toParcelObject(parcelsArray, prevParcels) {
-  return connectParcels(
-    parcelsArray,
-    parcelsArray.reduce((map, parcel) => {
-      map[parcel.id] = normalizeParcel(parcel, prevParcels[parcel.id])
-      return map
-    }, {}),
-    prevParcels
-  )
+  const parcelObject = {}
```
Comment (nachomazzara) [inline shared/parcel.js]: Is it wrong if we call it `parcelsObject`?

File: shared/parcel.js (hunk 1 above)
Comment (nachomazzara) [reply shared/parcel.js]: Yep, with the new commit is fixed 👍

File: shared/parcel.js (hunk 2 above)
Comment (cazala) [reply shared/parcel.js]: the function is called `toParcelObject` that's why i used `parcelObject`. But for consistency I think we could rename `parcelsArray` to `parcelArray`, right?

Comment (nachomazzara) [reply shared/parcel.js]: I think that would be better to rename the function to `toParcelsObject` its return an object with all the parcels as an object. And also use `parcelsObject` and what you say `parcelArray`

Clean version:

`toParcelObject` to `toParcelsObject`
`parcelObject` to `parcelsObject`
`parcelsArray` to `parcelArray`

**Pull request merged**